python/ray/_private/services.py

```diff
@@ -1344,24 +1340,32 @@ def start_api_server(
         )
 
         # Retrieve the dashboard url
-        gcs_client = GcsClient(address=gcs_address, cluster_id=cluster_id_hex)
```

Comment:
A small improvement: no need to create a new GcsClient but can use a passed in one.